crates/rust-analyzer/src/bin/args.rs

```diff
@@ -47,11 +49,16 @@ FLAGS:
     -vv, --spammy
     -q,  --quiet      Set verbosity
 
-    --log-file <PATH> Log to the specified filed instead of stderr
+    --log-file <PATH> Log to the specified file instead of stderr
+    --no-buffering    Flush log records to the file immediately
```

Comment:
```suggestion
    --no-log-buffering  Flush log records to the file immediately
```

Reply:
> But maybe I am missing smth windows-specific here...

Whoops, my bad. Totally forgot that I minimized the pipe buffer. No lags with stderr at all, so I made the stderr flush unconditional.